test/e2e/utils.go

```diff
@@ -221,3 +266,17 @@ func containerFound(containers []*runtimeapi.Container, containerID string) bool
 	}
 	return false
 }
+
+func checkLog(podConfig *runtimeapi.PodSandboxConfig, logPath string, expectedContext string) {
```

Comment:
Could we also verify log contents? the format of log file is same with docker's json-log.